src/sssom/util.py

```diff
@@ -1101,6 +1103,11 @@ def is_curie(string: str) -> bool:
 
 def is_iri(string: str) -> bool:
     """Check if the string is an IRI."""
+    warnings.warn(
+        "Use :meth:`curies.Converter.is_uri` via a well-defined Converter instance",
+        DeprecationWarning,
+        stacklevel=2,
+    )
     return validators.url(string)
```

Comment:
Maybe replace this by `curies.converter.is_uri()` to ensure consistent behaviour (not removing the method, wrapping the converter method)?

Reply:
it's actually only used in one place, where we need global checking for URI-ness. I just made an update to make this all more clear

Reply:
Ok, thank you!